### DISCUSSION TITLE
Updating to the new nerd font has shrunk the arrow prompt

### DISCUSSION THREAD

**Discussion by Nukiloco** [starship/starship#5215] [Q&A]
Before:
![Selection_013](https://github.com/starship/starship/assets/16618467/d4359714-16cf-484e-859c-33e4dcb25efe)

After:
![Selection_012](https://github.com/starship/starship/assets/16618467/2ec65b2e-013b-48dd-ad52-b6dc564ef293)

Just wondering why has this happened? Is this intentional? Would there be anyway to get this back to how it looked like before?

**Comment by mickimnet** [comment]
What terminal and what font are you using?

**Reply by Nukiloco** [reply]
I'm using alacritty currently as my terminal.

**Comment by davidkna** [comment]
I think I read something about some symbols no longer being rendered too large in the nf changelogs.
